diff -up fcron-3.3.2/fcronsighup.c.omv~ fcron-3.3.2/fcronsighup.c
--- fcron-3.3.2/fcronsighup.c.omv~	2025-01-01 23:23:09.271776481 +0100
+++ fcron-3.3.2/fcronsighup.c	2025-01-01 23:23:24.274982700 +0100
@@ -98,7 +98,7 @@ sig_daemon(void)
     if (max_delay_s > 0) {
         time_t now_epoch = 0;
         int delay_s = 0;
-        time_t *target_time_epoch = NULL;
+        time_t target_time_epoch;
         struct tm *target_time_tm = NULL;
         FILE *fp = NULL;
         int fd = 0;
