diff -up fcron-3.3.2/Makefile.in.1~ fcron-3.3.2/Makefile.in
--- fcron-3.3.2/Makefile.in.1~	2024-08-25 00:45:47.000000000 +0200
+++ fcron-3.3.2/Makefile.in	2025-01-01 23:11:50.215850639 +0100
@@ -152,7 +152,6 @@ ifneq ($(SYSTEMD_DIR), no)
 	if test ! -d $(DESTDIR)$(SYSTEMD_DIR); then $(INSTALL) -m 755 -d $(DESTDIR)$(SYSTEMD_DIR) ; fi
 	$(INSTALL) -m 644 script/fcron.init.systemd $(DESTDIR)$(SYSTEMD_DIR)/fcron.service
 	$(INSTALL) -m 644 script/fcron.init.systemd.reboot $(DESTDIR)$(SYSTEMD_DIR)/fcronreboot.service
-	systemctl daemon-reload
 endif
 
 install-staged: all
@@ -160,7 +160,7 @@ install-staged: all
 # needed by boot-install script :
 	echo `fcron -V 2>&1 | grep "^fcron "` > PREVIOUS_VERSION && chmod a+rw PREVIOUS_VERSION
 
-	$(SRCDIR)/script/user-group $(USERNAME) $(GROUPNAME) $(ANSWERALL) $(SRCDIR)
+	# $(SRCDIR)/script/user-group $(USERNAME) $(GROUPNAME) $(ANSWERALL) $(SRCDIR)
 
 # check if the directories we use exist, and if not, create them
 	if test ! -d $(DESTDIR)$(DESTSBIN); then $(INSTALL) -m 755 -d $(DESTDIR)$(DESTSBIN) ; fi
@@ -186,45 +186,45 @@ endif
 perms: install-staged strip
 # Note : we don't use "chown user:group file" because some systems use ":"
 #        and others "." as separator.
-	chown $(ROOTNAME) $(DESTDIR)$(DESTSBIN)
-	chgrp $(ROOTGROUP) $(DESTDIR)$(DESTSBIN)
-	chown $(ROOTNAME) $(DESTDIR)$(DESTBIN)
-	chgrp $(ROOTGROUP) $(DESTDIR)$(DESTBIN)
-	chown $(ROOTNAME) $(DESTDIR)$(ETC)
-	chgrp $(ROOTGROUP) $(DESTDIR)$(ETC)
-	chown $(ROOTNAME) $(DESTDIR)$(FIFODIR)
-	chgrp $(ROOTGROUP) $(DESTDIR)$(FIFODIR)
-	chown $(ROOTNAME) $(DESTDIR)$(PIDDIR)
-	chgrp $(ROOTGROUP) $(DESTDIR)$(PIDDIR)
+#	chown $(ROOTNAME) $(DESTDIR)$(DESTSBIN)
+#	chgrp $(ROOTGROUP) $(DESTDIR)$(DESTSBIN)
+#	chown $(ROOTNAME) $(DESTDIR)$(DESTBIN)
+#	chgrp $(ROOTGROUP) $(DESTDIR)$(DESTBIN)
+#	chown $(ROOTNAME) $(DESTDIR)$(ETC)
+#	chgrp $(ROOTGROUP) $(DESTDIR)$(ETC)
+#	chown $(ROOTNAME) $(DESTDIR)$(FIFODIR)
+#	chgrp $(ROOTGROUP) $(DESTDIR)$(FIFODIR)
+#	chown $(ROOTNAME) $(DESTDIR)$(PIDDIR)
+#	chgrp $(ROOTGROUP) $(DESTDIR)$(PIDDIR)
 
 # change spool dir mode
-	chown $(USERNAME) $(DESTDIR)$(FCRONTABS)
-	chgrp $(GROUPNAME) $(DESTDIR)$(FCRONTABS)
+#	chown $(USERNAME) $(DESTDIR)$(FCRONTABS)
+#	chgrp $(GROUPNAME) $(DESTDIR)$(FCRONTABS)
 	chmod 770 $(DESTDIR)$(FCRONTABS)
 
-	chown $(ROOTNAME) $(DESTDIR)$(DESTSBIN)/fcron
-	chgrp $(ROOTGROUP) $(DESTDIR)$(DESTSBIN)/fcron
+#	chown $(ROOTNAME) $(DESTDIR)$(DESTSBIN)/fcron
+#	chgrp $(ROOTGROUP) $(DESTDIR)$(DESTSBIN)/fcron
 	chmod 710 $(DESTDIR)$(DESTSBIN)/fcron
 
-	chown $(USERNAME) $(DESTDIR)$(DESTBIN)/fcrontab
-	chgrp $(GROUPNAME) $(DESTDIR)$(DESTBIN)/fcrontab
+#	chown $(USERNAME) $(DESTDIR)$(DESTBIN)/fcrontab
+#	chgrp $(GROUPNAME) $(DESTDIR)$(DESTBIN)/fcrontab
 	chmod $(BINMODE) $(DESTDIR)$(DESTBIN)/fcrontab
 
-	chown $(ROOTNAME) $(DESTDIR)$(DESTBIN)/fcronsighup
-	chgrp $(GROUPNAME) $(DESTDIR)$(DESTBIN)/fcronsighup
+#	chown $(ROOTNAME) $(DESTDIR)$(DESTBIN)/fcronsighup
+#	chgrp $(GROUPNAME) $(DESTDIR)$(DESTBIN)/fcronsighup
 	chmod $(BINMODESIGHUP) $(DESTDIR)$(DESTBIN)/fcronsighup
 
 ifeq ($(FCRONDYN), 1)
-	chown $(USERNAME) $(DESTDIR)$(DESTBIN)/fcrondyn
-	chgrp $(GROUPNAME) $(DESTDIR)$(DESTBIN)/fcrondyn
+#	chown $(USERNAME) $(DESTDIR)$(DESTBIN)/fcrondyn
+#	chgrp $(GROUPNAME) $(DESTDIR)$(DESTBIN)/fcrondyn
 	chmod $(BINMODE) $(DESTDIR)$(DESTBIN)/fcrondyn
 endif
-	chown $(ROOTNAME) $(DESTDIR)$(ETC)/fcron.allow $(DESTDIR)$(ETC)/fcron.deny
-	chgrp $(GROUPNAME) $(DESTDIR)$(ETC)/fcron.allow $(DESTDIR)$(ETC)/fcron.deny
+#	chown $(ROOTNAME) $(DESTDIR)$(ETC)/fcron.allow $(DESTDIR)$(ETC)/fcron.deny
+#	chgrp $(GROUPNAME) $(DESTDIR)$(ETC)/fcron.allow $(DESTDIR)$(ETC)/fcron.deny
 	chmod 640 $(DESTDIR)$(ETC)/fcron.allow $(DESTDIR)$(ETC)/fcron.deny
 
-	chown $(ROOTNAME) $(DESTDIR)$(ETC)/fcron.conf
-	chgrp $(GROUPNAME) $(DESTDIR)$(ETC)/fcron.conf
+#	chown $(ROOTNAME) $(DESTDIR)$(ETC)/fcron.conf
+#	chgrp $(GROUPNAME) $(DESTDIR)$(ETC)/fcron.conf
 	chmod 640 $(DESTDIR)$(ETC)/fcron.conf
 ifeq ($(USEPAM), 1)
 	$(SRCDIR)/script/install-pam-conf $(SRCDIR) $(DESTDIR)$(ETC) $(ROOTNAME) $(ROOTGROUP) "$(INSTALL)"
